src/api/app/views/webui2/webui/project/_bottom_actions.html.haml

```diff
@@ -0,0 +1,21 @@
+%ul.list-inline.mb-0
+  - if (bugowners_mail.present? && !CONFIG['bugzilla_host'].nil?) || !User.current.is_nobody?
+    - if bugowners_mail.present? && !CONFIG['bugzilla_host'].nil?
+      = render partial: 'webui2/webui/project/bottom_actions/report_bug', locals: { bugowners_mail: bugowners_mail,
+                                                                                    project: project }
+    - if project.is_locked? && User.current.can_modify?(project, true)
+      = render partial: 'webui2/webui/project/bottom_actions/unlock_project', locals: { project: project }
+    - elsif User.current.can_modify?(project)
+      = render partial: 'webui2/webui/project/bottom_actions/modify_project', locals: { project: project,
```

Comment:
I don't really like the amount of nested partials (and locals) we have here. Can't we do something like

```ruby
if can_have_patchinfo?(project) # helper
  render :patchinfo
end
```
instead?

Also some of the instance variables we define in the controller and pass through every partial simple enough (not expensive) to fetch them directly if needed, e.g. [is_incident_project](
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/project_controller.rb#L786). That would also help making the code more readable.

Reply:
I also don't like the amount of nested partials and locals. By fetching them directly, you mean that you would make an AJAX call to fetch the value?

Reply:
I mean to drop `@is_incident_project` and call `@project.is_maintenance_incident?` where it is needed. The [method](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/project.rb#L493) doesn't seem to be expensive. So need for us to jump through hoops.

Reply:
Oh okay, I see what you mean.

Reply:
I just replaced `@is_incident_project` by project.is_maintenance_incident? and `@is_maintenance_project` by `project.is_maintenance?`. For the other instance variables, it's better to keep using them.

Reply:
I did the same in the side links, but in another PR: #6430.